src/view/components/wrappedField.jsx

```diff
@@ -99,7 +99,10 @@ class DecoratedInput extends React.Component {
 
     return (
       <Flex alignItems="end">
-        <CoreExtensionToolTipTrigger isDisabled={!error}>
+        <CoreExtensionToolTipTrigger
+          isDisabled={!error}
+          placement={errorTooltipPlacement}
```

Comment:
Since the last error tooltip rewrite, you couldn't choose the tooltip placement. This change is fixing this.